src/OpenTelemetry.Extensions.Hosting/Implementation/HostingExtensionsEventSource.cs

```diff
@@ -27,34 +26,10 @@ internal sealed class HostingExtensionsEventSource : EventSource
     {
         public static HostingExtensionsEventSource Log = new();
 
-        [NonEvent]
-        public void FailedInitialize(Exception ex)
+        [Event(1, Message = "OpenTelemetry TraverProvider and/or MeterProvider were not found in application services. SDK will remain disabled.", Level = EventLevel.Warning)]
+        public void SdkNotRegistered()
```

Comment:
Have dedicated events for TracerProvider and MeterProvider instead of using the same?

Reply:
Updated!